src/components/RouterBreadcrumbs.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Breadcrumbs from '@material-ui/lab/Breadcrumbs';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
+import { Link as RouterLink } from 'react-router-dom';
+import { Paper, withStyles } from '@material-ui/core';
+
+const styles = (theme) => ({
+  paper: {
+    padding: `${theme.spacing.unit}px ${theme.spacing.unit * 2}px`,
+  },
+  reactRouterHoverInherit: theme.reactRouterHoverInherit
+});
+
+/**
+ * Returns a list of breadcrumbs based on current path of navigation
+ * @param {string} pathname current URI pathname
+ * @return {Array} List of BreadcrumbItems
+ */
+const RouterBreadcrumbs = ({ pathname, classes }) => {
+  const breadcrumbItems = [];
+  const viewNames = ['browser', 'viewer', 'editor', 'repos'];
+  const keyBase = 'breadcrumb-item-';
+
+  const subDirectoriesArr = pathname.split('/');
```

Comment:
I recommend extracting this interesting business logic into a function, and then writing tests for it in the spec.js file.